doc/common/README.md

```diff
@@ -1,14 +1,13 @@
 [<< Back](../)
-# Cloud iNfrastructure Telco Taskforce
+# Anuket Project
 
 ## Table of Contents
-* [Introduction](#1.0)
-* [Approach](#4.0)
-* [Use Cases](#5.0)
-* [Roadmap and Releases](#6.0)
-* [CNTT Technical Policies and Transition Plan](#7.0)
-* [Other Policies](#9.0)
-* [Relevant Technologies](#8.0)
+* [Introduction](#Introduction)
+* [Use Cases](#Use-Cases)
+* [Roadmap and Releases](#Roadmap-and-Releases)
+* [Anuket Project Technical Policies and Transition Plan](#Anuket-Project-Technical-Policies-and-Transition-Plan)
+* [Other Policies](#Other-Policies)
+* [Relevant Technologies](#Relevant-Technologies)
```

Comment:
We actually don't need to maintain those anchors manually since GitHub Flavored Markup (and the Sphinx build on read the docs) supports implicit header anchors. See for instance a heading without an explicit anchor:
https://github.com/cntt-n/CNTT/tree/master/doc/ref_model#release-information

Reply:
@gkunz There are no manual anchors in the sections but the ToC needs to specify the auto-generated anchors to traverse from the ToC

Reply:
@pgoyal01 you are absolutely right - I mixed this up. Sorry!